File: lab_05/canchas_project/reservas/models/cancha.py
from django.db import models
from .empresa import Empresa

class Cancha(models.Model):

    TIPO_CHOICES = [
        ('futbol', 'Fútbol'),
        ('fulbito', 'Fulbito'),
        ('voley', 'Vóley'),
        ('otro', 'Otro'),
    ]

    empresa = models.ForeignKey(Empresa, on_delete=models.CASCADE, related_name="canchas")
    nombre = models.CharField(max_length=50)
    tipo = models.CharField(max_length=20, choices=TIPO_CHOICES)
    precio_hora = models.DecimalField(max_digits=8, decimal_places=2)

    def __str__(self):
        return f"{self.nombre} - {self.empresa.nombre}"

File: lab_06/scaffoldingCRUD/scaffoldingCRUD/Models/Cancha.cs
using System;
using System.Collections.Generic;

namespace scaffoldingCRUD.Models;

public partial class Cancha
{
    public int IdCancha { get; set; }

    public int IdEmpresa { get; set; }

    public string Nombre { get; set; } = null!;

    public string? Tipo { get; set; }

    public decimal? PrecioHora { get; set; }

    public virtual Empresa IdEmpresaNavigation { get; set; } = null!;
}
